pkgs/development/ocaml-modules/irmin/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, ocaml
+, astring, cstruct, fmt, hex, jsonm, logs, ocaml_lwt, ocamlgraph, uri
+}:
+
+buildDunePackage rec {
+  pname = "irmin";
+  version = "1.4.0";
+
+  minimumOCamlVersion = "4.03";
+
+  src = fetchFromGitHub {
```

Comment:
The released sources from <https://github.com/mirage/irmin/releases/download/1.4.0/irmin-1.4.0.tbz> have correct version information.

Reply:
Meaning I should directly use `fetchurl` ?

Reply:
Yes.